packages/lesswrong/server/notificationCallbacks.tsx

```diff
@@ -293,66 +67,80 @@ getCollectionHooks("Posts").editAsync.add(function PostsEditRunPostApprovedAsync
   }
 });
 
-export async function postsUndraftNotification(post: DbPost) {
-  //eslint-disable-next-line no-console
-  console.info("Post undrafted, creating notifications");
-
-  await postsNewNotifications(post);
-}
-
 function postIsPublic (post: DbPost) {
   return !post.draft && post.status === postStatuses.STATUS_APPROVED
 }
 
 // Export for testing
-/** Add new post notification callback on post submit */
+/** create notifications for a new post being published */
 export async function postsNewNotifications (post: DbPost) {
```

Comment:
This is the only function that changed in this file - the rest were moved to the new file